administration/src/bp-modules/applications/PreVerifiedEntitlementType.tsx

```diff
@@ -0,0 +1,36 @@
+import { GeneralJsonField, type JsonField, findValue } from './JsonFieldView'
+
+export const preVerifiedEntitlements = {
+  Juleica: 'blueCardJuleicaEntitlement',
+  Verein360: 'blueCardWorkAtOrganizationsEntitlement',
+  HonoredByMinisterPresident: 'goldenCardHonoredByMinisterPresidentEntitlement',
+} as const
+
+export type PreVerifiedEntitlementType = (typeof preVerifiedEntitlements)[keyof typeof preVerifiedEntitlements]
+
+type MatcherFn = (entitlement?: GeneralJsonField) => boolean
+
+const defaultMatcher: MatcherFn = entitlement => entitlement !== undefined
+
+const verein360Matcher: MatcherFn = entitlement => {
+  const entitlements = Array.isArray(entitlement?.value) ? entitlement.value : []
```

Comment:
```suggestion
  const entitlements = entitlement !== undefined && Array.isArray(entitlement.value) ? entitlement.value : []
```
🔧  Lacking undefined check